cluster-sync/external/README.md

```diff
@@ -0,0 +1,36 @@
+# External Kubernetes Provider
+
+This provider works with an existing Kubernetes cluster.  You'll want to configure your own
+Container Image Registry, we tie into the existing Make/Build workflow, but modify things
+such that:
+1. We build container images and push to the specified registry (instead of `registry:5000`)
+2. We generate the manifests with the provided DOCKER_PREFIX (uses default port)
+3. Uses your configured `kubectl` and deploys your build to the existing cluster
+
+# Building images and pushing to your registry
+
+```bash
+export DOCKER_PREFIX=index.docker.io/barney_rubble
```

Comment:
What is this `DOCKER_PREFIX`. I get 404 when using it in the browser.

Reply:
It's just an example how to build the images for your own.

Reply:
OK